src/services/api/handlers/getPackageActions.ts

```diff
@@ -94,7 +38,7 @@ export const getPackageActions = async (event: APIGatewayEvent) => {
     return response({
       statusCode: 200,
       body: {
-        actions: packageActionsForResult(userAttr, result),
+        actions: packageActionsForResult(userAttr, result._source),
```

Comment:
I wonder if we need the endpoint at all anymore? I figure if we have the data and user attributes available from the front-end, we can pass those in here and just nix the whole endpoint.

Reply:
I'll add a note to brian's backend refactor ticket.